dev/VSIX/Extension/Cpp/Dev17/WindowsAppSDK.Cpp.Extension.Dev17.csproj

```diff
@@ -49,19 +48,19 @@
     <Content Include="$(RepoRoot)LICENSE">
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
-    <Content Include="ExtensionPack.vsext">
+    <Content Condition="'$(Deployment)'=='Standalone'" Include="ExtensionPack.vsext">
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
     </ItemGroup>
   <ItemGroup>
     <Reference Include="System" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.0.0-previews-1-31325-273" ExcludeAssets="runtime">
+    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.0.0-previews-4-31709-430" ExcludeAssets="runtime">
```

Comment:
>Version="17.0.0-previews-4-31709-430" [](http://example.com/codeflow?start=59&length=37)

The two version updates in this change, we don't want them made in WindowsAppSDK.Cpp.Extension.Dev17.csproj and WindowsAppSDK.Cs.Extension.Dev17.csproj?

Reply:
Did you mean the Dev16 projects? We're not updating them because they are targeting Dev16, and so they're using the Dev16 VS SDK.

Reply:
Shoot. Yeah, I meant Dev16.
OK, got it.